common/app/googleAuth/OAuthLoginController.scala

```diff
@@ -21,7 +21,7 @@ trait OAuthLoginController extends BaseController with ImplicitControllerExecuti
   val authCookie = new AuthCookie(httpConfiguration)
 
   val LOGIN_ORIGIN_KEY = "loginOriginUrl"
-  val ANTI_FORGERY_KEY = "antiForgeryToken"
+  val ANTI_FORGERY_KEY = "play-googleauth-session-id"
```

Comment:
Why did this need to change?

Reply:
Good question! The `oauth2Callback` method extracts a `GoogleAuthConfig` object from the request and checks it includes a sessionId. If it does it validates the user (step 6 of the anti-forgery process in the PR description): https://github.com/guardian/frontend/blob/2d13267310bf0cde1618b8b2f2f64859e75731e5/common/app/googleAuth/OAuthLoginController.scala#L57-L70. 

This PR https://github.com/guardian/play-googleauth/pull/52/files#diff-db204c4600ba54edb3c5c4d046d1521b33ec84ba8f22794a506ddac68fb24600L41 modified the `GoogleAuthConfig` object to have an `AntiForgeryChecker` instead of an `antiForgeryKey` as a member variable. This had an effect on how we can access the session id: https://github.com/guardian/play-googleauth/pull/52/files#diff-db204c4600ba54edb3c5c4d046d1521b33ec84ba8f22794a506ddac68fb24600R92-R95.